Automattic/jetpack
# Phan: Add phan configs to all php projects

## Proposed changes:

Per pdWQjU-IH-p2, we're going to use Phan for PHP static analysis.

This commit adds Phan configs to all the projects, and to new projects created with `jetpack generate`.

It also adds a global production-exclude for the `.phan/` directory, as we'll never want that in mirror repos.

### Other information:


## Jetpack product discussion


pdWQjU-IH-p2

## Does this pull request change what data or activity we track or use?


No

## Testing instructions:






* Make sure the build artifact does not contain any `.phan/` directories. Other than that, there's nothing to test here.
